src/kvstore/RocksEngineConfig.cpp

```diff
@@ -71,9 +72,38 @@ DEFINE_int32(num_compaction_threads, 0,
 DEFINE_int32(rate_limit, 0,
             "write limit in bytes per sec. The unit is MB. 0 means unlimited.");
 
+DEFINE_bool(enable_prefix_filtering, false,
+            "Whether or not to enable rocksdb's prefix bloom filter.");
+DEFINE_bool(enable_whole_key_filtering, true,
+            "Whether or not to enable the whole key filtering.");
+DEFINE_int32(prefix_length, 12,
+            "The prefix length, default value is 12 bytes(PartitionID+VertexID).");
+
 namespace nebula {
 namespace kvstore {
 
+class GraphPrefixTransform : public rocksdb::SliceTransform {
+private:
+    size_t prefix_len_;
```

Comment:
Please pay attention to the way the variables are named, keep it consistent with our code, thx.

Reply:
Changed to prefixLen